### ISSUE TITLE
Desktop frontend layout (start page)

### ISSUE THREAD

**Issue by Ciavarella** [chas-academy/centaurs-06-imdb-clone#91]

**Comment by axelolsson** [comment]
17 days since this was opened... Progress?

**Comment by Ciavarella** [comment]
Redid everything, will show you progress soon @axelolsson

**Comment by axelolsson** [comment]
@Ciavarella Alright, I get it. But let's play with the idea that I wasn't a product owner that was very intrusive and picky with my project, and let's also pretend that I have no idea about development.

How do you think a client would react if you, or someone on your team, sat down on a demo at the end of a sprint and said "yeah, we redid this everything".

Now I'm sure there's a perfectly reasonable explanation for doing this, but you need to remember to firstly communicate _why_ this has to be done, and then - if the product owner deems it viable, and only then, are you allowed to redo stuff.

Again, I know it's picky but I want to prepare you guys for the real world where passion and attention for detail needs to be explained – in layman's terms, so that everyone can understand it - especially the person paying your salary.
